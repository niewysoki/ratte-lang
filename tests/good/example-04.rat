fn Main () -> Void {
  Println("napis");
  Println(ShowInt(5));
  Println(ShowBoolean(True));
  Println(Concat("liczba: ", ShowInt(5)));
  return;
}