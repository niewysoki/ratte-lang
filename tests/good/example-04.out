napis
5
True
liczba: 5
